backend/src/main/java/com/tourneynizer/tourneynizer/dao/MatchDao.java

```diff
@@ -34,14 +40,14 @@ public void insert(Match match, User user) throws EmailTakenException, SQLExcept
             this.jdbcTemplate.update(connection -> {
                 PreparedStatement preparedStatement = connection.prepareStatement(sql, new String[]{"id"});
                 preparedStatement.setLong(1, match.getTournamentId());
-                preparedStatement.setLong(2, match.getTeam1_id());
-                preparedStatement.setLong(3, match.getTeam2_id());
-                preparedStatement.setLong(4, match.getScore1());
-                preparedStatement.setLong(5, match.getScore2());
+                setNullable(preparedStatement, 2, match.getTeam1Id());
+                setNullable(preparedStatement, 3, match.getTeam2Id());
+                setNullable(preparedStatement, 4, match.getScore1());
+                setNullable(preparedStatement, 5, match.getScore2());
```

Comment:
A few things here:
* Why don't you store child 1 and child 2 in the database? Do you have some way of keeping track of it through match order?
* Please don't forget to remove court number at some point please.

Reply:
- Forgot, will fix
- Soon